packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -121,9 +121,12 @@ interface FluidDataStoreMessage {
 /**
  * Represents the context for the store. This context is passed to the store runtime.
  */
-export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidDataStoreContextEvents> implements
-    IFluidDataStoreContext,
-    IDisposable {
+export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidDataStoreContextEvents>
+    implements IFluidDataStoreContext, IDisposable
+{
+    public readonly logger: ITelemetryLogger;
+    private readonly contextLogger: ITelemetryLogger;
```

Comment:
Considering the mixin logger/config work that's pending, I would not name this using the `context` prefix as it would be confusing and/or redundant (as we're in a `context` class). Perhaps parentLogger and logger would be an alternative?

Reply:
Or baseLogger and logger.